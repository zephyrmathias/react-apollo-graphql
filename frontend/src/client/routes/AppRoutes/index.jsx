import React from 'react';
import { Switch, Route } from 'react-router-dom';
import HomePage from '../../pages/HomePage';
import ChannelPage from '../../pages/ChannelPage';

const AppRoutes = () => (
  <Switch>
    <Route path="/" exact component={HomePage} />
    <Route path="/channels" exact component={ChannelPage} />
  </Switch>
);

export default AppRoutes;
