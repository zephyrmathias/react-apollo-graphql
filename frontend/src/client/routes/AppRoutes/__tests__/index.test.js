import React from 'react';
import { shallow } from 'enzyme';
import AppRoutes from '../index';

describe('routes: <AppRoutes />', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<AppRoutes />);
    expect(wrapper).toMatchSnapshot();
  });
});
