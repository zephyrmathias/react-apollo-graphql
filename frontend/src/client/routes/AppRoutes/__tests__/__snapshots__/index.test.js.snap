// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes: <AppRoutes /> should render correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</Switch>
`;
